bin/plugins.json

```diff
@@ -99,6 +99,16 @@
     "require_install": "true",
     "category": "elasticsearch"
   },
+  "odfe-cli": {
+    "git": "opendistro-for-elasticsearch/odfe-cli",
+    "zip": "downloads/elasticsearch-clients/opendistro-cli",
+    "deb": "none",
+    "rpm": "none",
+    "windows": "none",
+    "cutversion": "1.0.0",
+    "require_install": "false",
+    "category": "elasticsearch"
+  },
```

Comment:
This is a client category not elasticsearch

Reply:
Will make the change.

Reply:
Completed the change.